blog-cse/2023-12-06-application-update.md

```diff
@@ -23,7 +23,7 @@ For full details, see the [Automation Service documentation](/docs/platform-serv
 
 ### Minor Changes and Enhancements
 
-* [New] Entity Groups now support second-level unnormalized attributes (`fields.*attribute*.*attribute*`).
+* [New] Entity Groups now support second-level unnormalized attributes **(fields.*attribute*.*attribute*)**.
```

Comment:
@pkazmir-sumo, the way we usually show this kind of thing is: `fields.<attribute>.<attribute>`